workflows/mcmicro.nf

```diff
@@ -34,9 +36,18 @@ workflow MCMICRO {
     ch_markersheet // channel: markersheet read in from --marker_sheet
 
     main:
-
     ch_versions = Channel.empty()
     ch_multiqc_files = Channel.empty()
+
+    metadata = UPDATE_FROM_OME(ch_samplesheet, ch_markersheet)
+    ch_versions.mix(metadata.versions)
```

Comment:
the result of the `mix` operator is a new channel that needs to be assigned somewhere. In this case you want to assign it back to `ch_versions`.
```suggestion
    ch_versions = ch_versions.mix(metadata.versions)
```